options.rb

```diff
@@ -0,0 +1,133 @@
+module STUDENTTEACHER
+  def student_option()
+    print 'Age '
+    age = gets.chomp.to_i
+    print 'Name '
+    name = gets.chomp!
+    print 'Has the parent permission [Y/N]'
+    inputpermission = gets.chomp!
+
+    case inputpermission.downcase
+    when 'y'
+      parent = true
+    when 'n'
+      parent = false
+    else
+      puts 'Error selectc the correct option'
+    end
+    @students << Student.new(age, name, ClassRoom.new(''), '', parent_permission: parent)
+    @people << Student.new(age, name, ClassRoom.new(''), '', parent_permission: parent)
+    puts 'Student added successfully'
+  end
+
+  def teacher_option()
+    print 'Age '
+    age = gets.chomp.to_i
+    print 'Name '
+    name = gets.chomp!
+    print 'Especiality '
+    speciality = gets.chomp!
+
+    @teachers << Teacher.new(age, name, speciality)
+    @people << Teacher.new(age, name, speciality)
+    puts 'Techaer added successfully'
+  end
+end
+
+module OPTIONS
+  include STUDENTTEACHER
+  def option1()
+    if @books.empty?
+      puts 'There are not books added'
+    else
+      puts 'Books'
+      @books.each do |element|
+        puts "Title  #{element.title} Author #{element.author}"
+      end
+    end
+  end
+
+  def option2()
+    if @teachers.empty? && @students.empty?
+      puts 'There are not people added'
+    else
+      @students.each do |element|
+        agep = element.age.to_s
+        puts "[Student: ] Name: #{element.name} ID:  #{element.id} Age: #{agep}"
+      end
+      @teachers.each do |element|
+        agep = element.age.to_s
+        puts "[Teacher: ] Name: #{element.name} ID: #{element.id} Age: #{agep}"
+      end
+    end
+  end
+
+  def option3()
+    puts "Do you want to create a student (1) or a teacher (2)? [
+    Input the number]: "
+    @personoption = gets.chomp.to_i
+
+    case @personoption
+    when 1
+      student_option
+    when 2
+      teacher_option
+    else
+      puts 'person number incorrect choose the options again'
+    end
+  end
+
+  def option4()
+    print 'Title '
+    title = gets.chomp!
+    print 'Author '
+    author = gets.chomp!
+    @books << Book.new(title, author)
+    puts 'Book created successfully'
+  end
+
+  def option5()
+    puts 'Select a book frmo the following list by number'
+    @books.each_with_index do |element, index|
+      puts "#{index}) Title  #{element.title} Author #{element.author}"
+    end
+    bookselected = gets.chomp.to_i
+    puts 'Select a person from the following list by number (not id)'
+    @people.each_with_index do |element, index|
+      agep = element.age.to_s
+      typeo = element.class.to_s
+      puts "#{index}) #{typeo} Name: #{element.name} ID:  #{element.id} Age: #{agep}"
+    end
+    personselected = gets.chomp.to_i
+    puts 'Set the current date'
+    date = gets.chomp!
```

Comment:
**[OPTIONAL]**

- Let's also give a little format of how the users should input the date, as this helps users know how to add the date easily. Currently, there's no date format, which makes it unclear how they should write the date. let's improve on this to improve the user experience interacting with the app.